### DISCUSSION TITLE
Mode and Callsign as Hashtag in Mastodon-Toots?

### DISCUSSION THREAD

**Discussion by dg9vh** [magicbug/Cloudlog#2302] [Ideas / Feature Requests]
Hi,
actually we have #hamr and #cloudlog as tags in toots (and tweets), how about adding mode and callsign of worked station as hashtag, too?

73 de Kim
DG9VH

**Comment by int2001** [comment]
Hmm.
Not sure. Some hams use mastodon/twitter, so a mention (@) would be more helpful then.
One ist able edit the tweet/toot before posting it.

Mode is a thing that could(!) be interesting as hashtag...

**Reply by dg9vh** [reply]
well, @-mention would be easy on Twitter (but who is using this???) on mastodon we have different host-parts so it would not be easy to find the right one... Editing is right, but this produces "overhead" ;-)

**Comment by magicbug** [comment]
Don't see the point of adding the callsign as a hashtag perhaps the mode for filtering but even then its just repeated data.

**Reply by dg9vh** [reply]
@magicbug what I mean (I should mention it directly when having such ideas, I see it was not clearly):

actually we have:
Just worked W1ABC in United States Of America (Gridsquare: FN02 / distance: 6321.2 km) on 15m using FT8 #hamr #cloudlog

My Idea now:

Just worked #W1ABC in United States Of America (Gridsquare: FN02 / distance: 6321.2 km) on 15m using FT8 #hamr #cloudlog  

==> so just adding the # before the callsign to have this also as a filter-criteria for the searches...

**Reply by magicbug** [reply]
Still not convinced
